### PR TITLE
Responsive landing

### PR DESCRIPTION
**Pull request by Ladu-Lumori**

### CODE REVIEW

File: apps/web/pages/landing.tsx [hunk 1]
```diff
@@ -71,310 +71,443 @@ const process = [
...
-        alignItems="center"
-        justifyContent="center"
-        height="100vh"
-      >
-        <Heading textAlign="center">
-          Lets turn every onboarding into a meaningful, structured, and truly
-          welcoming experience.
-        </Heading>
+    // <Flex direction="column">
+    //   <Box height="100vh">
```
Comment (Emmanuel-Melon) [inline apps/web/pages/landing.tsx:345]: The commented code?

Comment (Ladu-Lumori) [reply apps/web/pages/landing.tsx:345]: getting rid of it

Comment (Ladu-Lumori) [reply apps/web/pages/landing.tsx]: gotten rid of it

**Review by Emmanuel-Melon — APPROVED** [review]
LGTM

**Pull request merged**